flink-dist/src/main/flink-bin/bin/pyflink-shell.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+bin=`dirname "$0"`
+bin=`cd "$bin"; pwd`
+
+. "$bin"/config.sh
+
+FLINK_CLASSPATH=`constructFlinkClassPath`
+
+PYFLINK_INTERPRETER="${PYFLINK_INTERPRETER:-"python"}"
+
+# So that python can find out Flink's Jars
+export FLINK_BIN_DIR=$FLINK_BIN_DIR
+export FLINK_ROOT_DIR
+
+# Add pyflink & py4j to PYTHONPATH
+opt=`dirname "$0"`
+opt=`cd "$opt"/../opt; pwd`
+export PYTHONPATH="$opt/python/pyflink.zip:$PYTHONPATH"
+PY4J_ZIP=`echo "$opt/python/py4j-*-src.zip"`
```

Comment:
The star character in double quote won‘t expand, how about remove the double quotes or only quotes the "$opt"?